.connector-store/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Sift Connector",
+  "owner": "WSO2",
+  "category": "Adaptive Authentication Function",
+  "documentationUrl": "https://github.com/wso2-extensions/identity-fraud-detection-sift/tree/main/docs",
+  "description": "The Sift connector enables the invocation of the Sift API within the adaptive authentication script, providing risk-based authentication (RBA) to enhance security and prevent fraud.",
+  "status": "Active",
+  "labels": ["RBA"],
```

Comment:
What does RBA mean? I think its not a widely used term.

Reply:
This is an already existing tag. Risk based authentication